### PR TITLE
User story34

### PR DESCRIPTION
**Pull request by zachlp2016**
**### What does this PR do?**

Adds spec for adding merchant show page.
Adds testing that the merchant sees profile data.
Adds testing that the merchant doesn't see an edit profile link.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -21,8 +21,8 @@
     resources :orders, only: [:index]
   end
 
-  get '/dashboard', to: 'merchants#show'
 
+  get '/dashboard', to: 'merchants#show'
   get '/merchants', to: 'merchants#index' # a link to see all merchants ("/merchants") -- User Story 2, Visitor Navigation
```
Comment (VinceCarollo) [inline config/routes.rb:27]: Good Organization.

**Review by VinceCarollo — APPROVED** [review]
Looks Good. Nice work.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -63,7 +63,7 @@ def password_confirmation
     end
   end
 
-  def strong_params
+  def user_params
```
Comment (WHomer) [inline app/controllers/users_controller.rb:66]: nice change.

**Review by WHomer — APPROVED** [review]
Pretty fondue of this PR.

**Comment by WHomer** [comment]
#92

**Pull request merged**